system-variables.md

```diff
@@ -749,95 +783,105 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
 > Since v5.0, this variable is deprecated. Instead, use [`tidb_executor_concurrency`](#tidb_executor_concurrency-new-in-v50) for setting.
 
 - Scope: SESSION | GLOBAL
-- Default value: -1
+- Default value: `-1`
+- Range: `[1, 2147483647]`
 - This variable is used to set the concurrency of the `index lookup join` algorithm.
 - A value of `-1` means that the value of `tidb_executor_concurrency` will be used instead.
 
 ### tidb_index_lookup_size
 
 - Scope: SESSION | GLOBAL
-- Default value: 20000
+- Default value: `20000`
+- Range: `[1, 2147483647]`
 - This variable is used to set the batch size of the `index lookup` operation.
 - Use a bigger value in OLAP scenarios, and a smaller value in OLTP scenarios.
 
 ### tidb_index_serial_scan_concurrency
 
 - Scope: SESSION | GLOBAL
-- Default value: 1
+- Default value: `1`
+- Range: `[1, 2147483647]`
 - This variable is used to set the concurrency of the `serial scan` operation.
 - Use a bigger value in OLAP scenarios, and a smaller value in OLTP scenarios.
 
 ### tidb_init_chunk_size
 
 - Scope: SESSION | GLOBAL
-- Default value: 32
-- Range: 1 - 32
+- Default value: `32`
+- Range: `[1, 32]`
 - This variable is used to set the number of rows for the initial chunk during the execution process.
 
 ### tidb_isolation_read_engines <span class="version-mark">New in v4.0</span>
 
 - Scope: SESSION
-- Default value: tikv, tiflash, tidb
+- Default value: `tikv,tiflash,tidb`
 - This variable is used to set the storage engine list that TiDB can use when reading data.
 
 ### tidb_low_resolution_tso
 
 - Scope: SESSION
-- Default value: OFF
+- Default value: `OFF`
 - This variable is used to set whether to enable the low precision TSO feature. After this feature is enabled, new transactions use a timestamp updated every 2 seconds to read data.
 - The main applicable scenario is to reduce the overhead of acquiring TSO for small read-only transactions when reading old data is acceptable.
 
 ### tidb_max_chunk_size
 
 - Scope: SESSION | GLOBAL
-- Default value: 1024
-- Minimum value: 32
+- Default value: `1024`
+- Range: `[32, 2147483647]`
 - This variable is used to set the maximum number of rows in a chunk during the execution process. Setting to too large of a value may cause cache locality issues.
 
 ### tidb_max_delta_schema_count <span class="version-mark">New in v2.1.18 and v3.0.5</span>
 
 - Scope: GLOBAL
-- Default value: 1024
+- Default value: `1024`
+- Range: `[100, 16384]`
 - This variable is used to set the maximum number of schema versions (the table IDs modified for corresponding versions) allowed to be cached. The value range is 100 ~ 16384.
 
 ### tidb_mem_quota_apply_cache <span class="version-mark">New in v5.0</span>
 
 - Scope: SESSION | GLOBAL
-- Default value: 32 MB
+- Default value: `33554432`
```

Comment:
I think `32 MB` is easier to read. Note that `32 MB` is sort-of wrong as it is `32 MiB`. But as `SET GLOBAL...` doesn't accept values like this and `SHOW GLOBAL VARIABLES` doesn't do that either it is probably ok to do this.

```suggestion
- Default value: `33554432` (32 MiB)
```

Reply:
I think this is the right way to do it (human size in brackets). As you note the previous example is problematic because `SET GLOBAL` won't accept it. I will work on it.